README.md

```diff
@@ -0,0 +1,80 @@
+<head>
+  <meta name="Momento Laravel cache driver" content="Taggable Momento serverless cache driver for Laravel">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
```

Comment:
we should be able to use the README generator for this.  As long as you set the "type" to "other" then it won't require all of the sections that an SDK README requires, but you will get all of the header and badge stuff w/o having to manage it by hand.